src/DataFixtures/NewsFixtures.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\News;
+use DateTime;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+
+class NewsFixtures extends Fixture
+{
+
+    public function load(ObjectManager $manager)
+    {
+
+        // create 3 news!
+        for ($i = 0; $i < 3; $i++) {
```

Comment:
```suggestion
        for ($i = 1; $i <= 3; $i++) {
```
и не придется вводить доп переменную numberNews